src/ROUTES.ts

```diff
@@ -55,6 +55,8 @@ const ROUTES = {
 
     SEARCH_ADVANCED_FILTERS_CATEGORY: 'search/filters/category',
 
+    SEARCH_ADVANCED_FILTERS_CARD: 'search/filters/card',
```

Comment:
remove emptyline before this route, as Jakub is removing all the empty lines within search routes definitions.